corporate/lib/stripe.py

```diff
@@ -1170,6 +1189,50 @@ def set_required_plan_tier(self, required_plan_tier: int) -> str:
             plan_tier_name = CustomerPlan.name_from_tier(new_plan_tier)
         return f"Required plan tier for {self.billing_entity_display_name} set to {plan_tier_name}."
 
+    def create_fixed_price_plan(self, fixed_price: int) -> str:
+        customer = self.get_customer()
+        if customer is None:
+            customer = self.update_or_create_customer()
+
+        if customer.required_plan_tier is None:
+            raise SupportRequestError("Required plan tier should not be set to None")
+        required_plan_tier_name = CustomerPlan.name_from_tier(customer.required_plan_tier)
+
+        fixed_price_cents = fixed_price * 100
+        plan = get_current_plan_by_customer(customer)
```

Comment:
Let's call this variable `current_plan`, to avoid confusion with `fixed_price_plan`. Generally we should try to avoid variable names like this one that are potentially ambiguous if one isn't paying attention.